pootle/apps/pootle_project/forms.py

```diff
@@ -40,4 +46,20 @@ class Meta(object):
             model = TranslationProject
             fields = ('language', 'project')
 
+        def save(self, commit=True):
+            initialize_from_templates = False
+            tp = self.instance
+            if not tp.pk:
+                initialize_from_templates = tp.can_be_inited_from_templates()
```

Comment:
`tp.can_be_inited_from_templates()` already checks if `tp` has `pk` (or `id`, which is the same in this case). It also defaults to `False`. So there is no need to do these things here since that method does it for us. In other words, don't duplicate code.